components/styleguide/typography/ordered-lists/_ordered-list.css

```diff
@@ -0,0 +1,19 @@
+/*
+  Ordered Lists
+
+  Weight: 0
```

Comment:
Why all the elements inside typography are weight 0? I think the weight is for each level (typography in this case), not general per section (style guide).

Reply:
Are you sure? 

I put them as `0` so they will list in alphabetical order (inside this directory) - which seems to work.

Reply:
Exactly, you get alphabetical order and that's fine for components. But for an style guide I'd control the order: basic items like headings or basic text should be always first.

Reply:
They are now in this order:
body
headings
paragraph
link
unordered list
ordered list
blockquote